pkg/dockerfilegen/templates.go

```diff
@@ -2,19 +2,19 @@ package dockerfilegen
 
 import "embed"
 
-//go:embed dockerfile-templates/Default.dockerfile.tmpl
+//go:embed dockerfile-templates/Default.dockerfile.tmpl dockerfile-templates/rhel-9/*.tmpl
```

Comment:
```suggestion
//go:embed dockerfile-templates/Default.dockerfile.tmpl dockerfile-templates/rhel-9/*
```

It should be dir input, otherwise pattern match with suffix doesn't work.